exercise_1/Graph.h

```diff
@@ -0,0 +1,29 @@
+//
+// Created by cdd on 15.07.19.
+//
+
+#ifndef EXERCISE_1_GRAPH_H
+#define EXERCISE_1_GRAPH_H
+
+#include <vector>
+
+struct Point {
```

Comment:
Point - hmm, is that a node or an edge?
In graph nodes are connected with edges. Nodes usually keep some data and edges tells you about the relation between the data.